src/ci/docker/host-x86_64/dist-x86_64-netbsd/build-netbsd-toolchain.sh

```diff
@@ -27,7 +27,10 @@ mkdir -p /x-tools/x86_64-unknown-netbsd/sysroot
 
 # URL=https://ci-mirrors.rust-lang.org/rustc
 
-SOURCE_URL=https://cdn.netbsd.org/pub/NetBSD/NetBSD-9.0/source/sets
+# FIXME: the archive URL is being used temporarily while the CDN is down.
+# Change back to the original URL once it is back up.
+# SOURCE_URL=https://cdn.netbsd.org/pub/NetBSD/NetBSD-9.0/source/sets
+SOURCE_URL=http://archive.netbsd.org/pub/NetBSD-archive/NetBSD-9.0/source/sets/src.tgz
```

Comment:
what about line 39? it's also using still using the cdn

Reply:
I added this too